setup.py

```diff
@@ -33,10 +33,10 @@ def __str__(self):
             get_pybind_include(),
             get_pybind_include(user=True),
             "cpp/external/eigen",
-            "cpp/include"
+            "cpp/include",
         ],
         language='c++',
-        extra_compile_args=["-O3", "-std=c++11"]
+        extra_compile_args=["-O3", "-std=c++11", "-march=native", "-fopenmp"]
```

Comment:
If we ever decide to build wheels, we'll need to be careful about what arguments we include. But for now this is fine.